src/DssSpell.t.base.sol

```diff
@@ -200,9 +200,9 @@ contract DssSpellTestBase is Config, DSTest, DSMath {
         // Test for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xE246C4ba65d95C2F902E39FBeb0047a67aB4F25a),        // populate with deployed spell if deployed
-            deployed_spell_created:         1642785985,        // use get-created-timestamp.sh if deployed
-            previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
+            deployed_spell_created:         0,        // use get-created-timestamp.sh if deployed
+            previous_spell:                 address(0xE246C4ba65d95C2F902E39FBeb0047a67aB4F25a),        // supply if there is a need to test prior to its cast() function being called on-chain.
```

Comment:
just a reminder that previous will be [available for execution in about 20 mins](https://vote.makerdao.com/executive?network=mainnet).